book/content/material/material_advanced.md

```diff
@@ -0,0 +1,273 @@
+---
+jupytext:
+  formats: ipynb,md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.17.3
+kernelspec:
+  display_name: festim-workshop
+  language: python
+  name: python3
+---
+
+# Advanced Material Features #
+
++++
+
+This tutorial introduces more advanced ways of handling materials in FESTIM. A key addition in FESTIM 2.0 is the ability to perform multi-species simulations, which often requires assigning distinct material properties to each species (see __[documentation](https://festim.readthedocs.io/en/v2.0-alpha.3/api/index.html#festim.material.Material)__ for more information). In some cases, users may also need to define custom material properties—for example, specifying a turbulence-dependent viscosity or user-defined diffusivity.
+
+Objectives:
+* Assign different material properties to different species
+* Solve a problem with custom material properties (e.g., diffusivity)
+
++++
+
+## Assigning different material properties in a multi-species simulation ##
+
+Consider the following 1D example that simulates the diffusion of protium, deuterium and tritium species:
+
+```{code-cell} ipython3
+:tags: [hide-input]
+
+import festim as F
+import numpy as np
+
+my_model = F.HydrogenTransportProblem()
+
+protium = F.Species("H")
+deuterium = F.Species("D")
+tritium = F.Species("T")
+my_model.species = [protium, deuterium, tritium]
+
+my_model.mesh = F.Mesh1D(np.linspace(0, 1, 100))
+
+left_surf = F.SurfaceSubdomain1D(id=1, x=0)
+right_surf = F.SurfaceSubdomain1D(id=2, x=1)
+```
+
+Here we define the different diffusivities for each species. The `mat` dictionary stores keys for each species' diffusion coefficient `D_0` and activation energy `E_D`. We define one domain for all species called `bulk` that serves as the volume for the material:
+
+```{code-cell} ipython3
+mat = F.Material(
+    D_0={protium: 1.0e-3, deuterium: 3.0e-3, tritium: 5.0e-3},  # example diffusivity values for H, D, T
+    E_D={protium: 0.0, deuterium: 0.0, tritium: 0.0},
+)
+
+# One bulk domain using this material
+bulk = F.VolumeSubdomain1D(id=1, borders=[0, 1], material=mat)
+
+my_model.subdomains = [bulk, left_surf, right_surf]
+```
+
+To illustrate how different species' diffusion properties affect the simulation, we set the same boundary condition for each species (1 on the left boundary, 0 on the right). Now we can run the simulation and look at the concentration profile for each species:
+
+```{code-cell} ipython3
+:tags: [hide-input]
```

Comment:
Same comment as above, if you explain this in the markdown cell, then don't hide it.

I think here you want to split this cell after `my_model.run()` and then only hide the post-processing cell